wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+// Exit if accessed directly
+if ( !defined( 'ABSPATH' ) ) exit;
+
+/**
+ * Enqueue child styles. Will be used to data via SESSION
+ */
+add_action( 'wp_enqueue_scripts', 'child_enqueue_styles' );
+function child_enqueue_styles()
+{
+    $parent_style = 'parent-style';
+    wp_enqueue_style( $parent_style, get_template_directory_uri() . '/style.css' );
+    wp_enqueue_style(
+        'twentytwenty-child-style',
+        get_stylesheet_directory_uri() . '/style.css',
+        array( $parent_style ),
+        wp_get_theme()->get( 'Version' )
+    );
+}
+
+/**
+ * Start user session for each individual user
+ */
+add_action( "init", "theme_start_session", 1 );
+function theme_start_session()
+{
+    if ( !session_id() )
+        session_start();
+}
+
+/** Woo Commerce Hooks */
+
+/**
+ * REMOVE Shipping and Billing Fields in Checkout
+ */
+add_filter( 'woocommerce_checkout_fields', 'custom_override_checkout_fields' );
+function custom_override_checkout_fields( $fields )
```

Comment:
I wonder if the WooCommerce plugin we're thinking to purchase will get rid of custom code like this?

Reply:
@bijanmmarkes & @jpham93 - I found this section which hopefully BEST EXPLAINS what the WC GF add-on will solve for us in terms of our development efforts and wondering if this plugin has been purchased yet? Please let me know once it's available so we can take advantage of the following concepts and workflows using...

- https://docs.woocommerce.com/document/woocommerce-gravity-forms-product-addons/#section-25
- https://woocommerce.com/products/gravity-forms-add-ons/

Also to help us best control form selection choices and workflow options checkout this...
- https://www.gravityforms.com/add-ons/chained-selects/
- https://docs.gravityforms.com/category/add-ons-gravity-forms/chained-selects/